### PR TITLE
Slight improvements for MADs "nearby_cells" mode

### PR DESCRIPTION
**Pull request by domizei385**
## Description
- group mons around stops when location is not known
- highlight when a location is inaccurate (nearby_cell) or very inaccurate (in L15 cell)
- limit session duration to 3 days, so that session are auto cleaned from redis

## Motivation and Context
better support for @ccev awesome work in MAD

## How Has This Been Tested?
- nothing formal also just ripped this out of a set of other changes, so no guarantees

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by kpresler** [comment]
There's something strange that appears to be going on with this PR where stuff disappears off of my map when it's active.  For example, when I'm running the PR:
image


If I switch back to `master`, I don't get spawns spread out, but everything else works about like I'd expect: 

image


Aside from confirming that it happens multiple times (ie, switch branch, `npm run build`, restart RM) I've basically not investigated, so this could well be a PEBKAC error.  I _have_ made some (local) changes to `app.py` and `quest.js` to support filtering Quests by region, but at least at first glance I don't see this being related.  Happy to post a diff of those changes if anyone wants, however.

**Comment by domizei385** [comment]
Thanks for the feedback.
I have no idea why some gyms would not load for you as I have not touched anything related to that. Did it throw any js exceptions on the console?

File: static/js/map/map.settings.js [hunk 1]
```diff
@@ -203,6 +204,18 @@ function initSettingsSidebar() {
...
 
+        $('#exclude-nearby-cell-switch').on('change', function () {
+            settings.excludeNearbyCells = this.checked
+            updateMap({ loadAllPokemon: true })
+            if (!this.checked) {
+                settings.showS2Cells = true
+                settings.showS2CellsLevel15 = true
+                Store.set('showS2Cells', true)
+                Store.set('showS2CellsLevel15', true)
+            }
```
Comment (na-ji) [inline static/js/map/map.settings.js:215]: Why would you display the S2 Cells with this setting?

File: static/js/map/map.settings.js [hunk 2]
```diff
@@ -203,6 +204,18 @@ function initSettingsSidebar() {
             Store.set('filterPokemonById', this.checked)
         })
 
+        $('#exclude-nearby-cell-switch').on('change', function () {
+            settings.excludeNearbyCells = this.checked
+            updateMap({ loadAllPokemon: true })
```
Comment (na-ji) [inline static/js/map/map.settings.js:209]: When toggling on this setting, the Pokémon won't disappear until you reload the page. When toggling off, they will appear right away. 
For this work in both cases,  you will also need to use the `updatePokemons` function.

**Review by na-ji — CHANGES_REQUESTED** [review]
Thanks for the PR. However the setting does not work all the time

**Comment by muckelba** [comment]
Works fine for me, but the two things mentioned by @na-ji are an issue for me too. I just commented the s2cells stuff for now.

**Review by na-ji — APPROVED** [review]

**Pull request merged**